### PR TITLE
Autocompletion support - Fixes #9

### LINKED ISSUE #9: Autocompletion Support
Hey,

I think it's possible and useful to add autocompletion to locus, I really like the idea, a true debugging assistant.

I'm a little busy with a few open-source projects and some other work, but I will definitely take a look, what do you think @alidavut?

### PR DESCRIPTION
**Pull request by mdibaiee**
About reading variables in scope, I couldn't find a solution, if you know a way to do that, I'd be happy to make it work.

### CODE REVIEW

**Comment by alidavut** [comment]
sorry @mdibaiee after my changes some conflicts appear, can you look at this again?

**Comment by mdibaiee** [comment]
Hey @alidavut I think it's ok now!

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
+var completer = require('./completer');
 global.locusModules = {};
 global.locusModules.deasync = require.resolve('deasync');
 global.locusModules.color = require.resolve('cli-color');
 global.locusModules.stackTrace = require.resolve('stack-trace');
+global.locusReadLine = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  completer: completer
```
Comment (alidavut) [inline src/index.js]: I think you should add this right after line 24, I moved readline into the function before.

**Comment by mdibaiee** [comment]
@alidavut Oh, sorry I was not aware of that change.

By moving readline there, as it's getting `eval`d inside target script, I had to move completer function inside listener. I tested it, it's working now.

**Comment by alidavut** [comment]
it works well, thanks @mdibaiee

**Pull request merged**